code/datums/diseases/scalerot.dm

```diff
@@ -0,0 +1,95 @@
+/datum/disease/scalerot
+	name = "Scale rot"
+	max_stages = 5
+	spread_text = "On contact"
+	spread_flags = CONTACT_GENERAL
+	cure_text = "Alcohol based anti-septics or vitamin superdoses."
+	cures = list("vitamin")
+	agent = "Reptilian Necrotizing Dermatitis"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	cure_chance = 10//scalerot, generally speaking, is not easy to cure, and requires lengthy decontamination procedures, and vitamin megadoses are pseudoscience.
+	desc = "A lethal bacteria that affects only scale based skin surfaces. Rots away scales. Commonly carried by various races, and naturally occurs on scaled races experiencing poor hygiene, moist skin, or excess humidty or moisture."
+	severity = DANGEROUS
+	longevity = 650
+
+/datum/disease/scalerot/stage_act()
+	..()
+	if(affected_mob.dna.species.id == "lizard"||affected_mob.dna.species.id == "fly") //Non-scaled species are not affected by the virus and merely carry it.
+		switch(stage)
+			if(1)
+				if(prob(40))
+					if(affected_mob.dna.species.id == "lizard")
```

Comment:
Still doesn't include the actual race ash lizards have, which is different, so they'll get no comment that something is wrong.